<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TcpServerClass</class>
 <widget class="QWidget" name="TcpServerClass">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>615</width>
    <height>806</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TcpServer</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>611</width>
     <height>801</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Address</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="editAddress">
        <property name="text">
         <string>127.0.0.1</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Port</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="editPort">
        <property name="text">
         <string>5500</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnListen">
        <property name="text">
         <string>Listen</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Recv:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="textRecv"/>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Send:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="textSend">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>150</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btnSend">
        <property name="text">
         <string>Send</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="tcpserver.qrc"/>
 </resources>
 <connections/>
</ui>
